code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -337,9 +337,7 @@
 
 /datum/spellbook_entry/summon/guns/Buy(var/mob/living/carbon/human/user,var/obj/item/weapon/spellbook/book)
 	feedback_add_details("wizard_spell_learned",log_name)
-	rightandwrong(0, user, 0)
-	book.uses += 1
-	active = 1
+	rightandwrong(0, user, 25)
 	playsound(get_turf(user),"sound/magic/CastSummon.ogg",50,1)
 	user << "<span class='notice'>You have cast summon guns and gained an extra charge for your spellbook.</span>"

```

Comment:
He's saying you didn't change the confirmation to say the wizard lost a
charge instead of gaining one.
On Jun 25, 2015 3:18 AM, "KorPhaeron" <notifications@github.com> wrote:

> In code/game/gamemodes/wizard/spellbook.dm
> <https://github.com/tgstation/-tg-station/pull/10157#discussion_r33231546>
> :
>
> > @@ -337,9 +337,7 @@
> >
> >  /datum/spellbook_entry/summon/guns/Buy(var/mob/living/carbon/human/user,var/obj/item/weapon/spellbook/book)
> >  	feedback_add_details("wizard_spell_learned",log_name)
> > -	rightandwrong(0, user, 0)
> > -	book.uses += 1
> > -	active = 1
> > +	rightandwrong(0, user, 25)
> >  	playsound(get_turf(user),"sound/magic/CastSummon.ogg",50,1)
> >  	user << "<span class='notice'>You have cast summon guns and gained an extra charge for your spellbook.</span>"
>
> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tgstation/-tg-station/pull/10157/files#r33231546>.
>